boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -121,31 +121,34 @@ the tool.
 Using JTAG
 ==========
 
-In addition to the SoC vendor and SDK, this board also requires a build of
-OpenOCD with patches supporting the SoC.  The source code and build
-instructions can be obtained from GitHub:
+As with much custom hardware, the ESP-32 moudules require patches to
+OpenOCD that are not upstream.  Espressif maintains their own fork of
+the project here.  By convention they put it in the `~/esp` next to the
+installations of their toolchain and SDK:
 
 .. code-block:: console
 
+   cd ~/esp
```

Comment:
Does the ~/exp folder already exist at this point, or do you need a `mkdir` before this to create it?